### DISCUSSION TITLE
Modern UI, Flat, Metro, Material?

### DISCUSSION THREAD

**Discussion by sjohnson1021** [AxionDrak/GameCube-Backup-Manager#93] [Announcements]
https://www.youtube.com/watch?v=PzP8mw7JUzI

https://www.youtube.com/watch?v=BtOEztT1Qzk&t=13s

Thoughts?

Rounded vs sharp? Dark,vs light (Both?) Use your blue? [#4d6082]

--for personal reference: https://stackoverflow.com/questions/1161459/wpf-application-using-a-global-variable

**Comment by laetemn** [comment]
Adding dark theme support would be nice in the current version of GCBM.
Modern Flat UI seems to be quite attractive, as well as blending better with Windows 10/11.
